README.md

```diff
@@ -80,13 +121,38 @@ export const getAnnouncementsByProjectGroup = async value => {
 ```
 This is in addition to the two default "get" functions created. 
 
-## Screenshots
+## Record Transformations
```

Comment:
After #19 is closed, will update with the pluralize + sort + filter functionality too

Reply:
Sounds good! I was imagining that all future PRs update the PR